docs/NEWSLOG.md

```diff
@@ -1,6 +1,21 @@
-## June 17, 2020
+## July 4, 2020
+### Wireguard VPN (ALPHA)
+Added Wireguard VPN (ALPHA).
+Wireguard can be deployed to enable for VPN access to your openHABian box when
```

Comment:
The one sentence per line thing does not appear to be stated anywhere in openhab-docs when I looked and in this case I don't particularly like the way in makes the text look so please change it.